Microsoft.Toolkit.Uwp.UI/Extensions/TextBoxRegEx/TextBoxRegex.Data.cs

```diff
@@ -34,7 +34,17 @@ public enum ValidationMode
             /// <summary>
             /// Update IsValid property with validation result and in case the textbox is not valid clear its value
```

Comment:
Update comment to specify the value is cleared on focus lost

Reply:
Done.